program.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+using namespace std;
+
+int main(){
+    cout << "This is the configuration file" << endl;
+    cout << "Version 1.0" << endl;
```

Comment:
It doesn't hurt to show version of the program here.

Reply:
Looks good